tests/testthat/test-github-connections.R

```diff
@@ -0,0 +1,187 @@
+context("GitHub connections")
+
+## set auth to TRUE if env var GITHUB_PAT is available (eg local testing)
+## set auth to FALSE otherwise (eg on travis, at this point)
+auth <- FALSE
```

Comment:
I set it to `TRUE` when I check locally, because I'm happy for my `GITHUB_PAT` to be used for tests of GitHub connectivity.

In theory, I change it back to `FALSE` every time I push, because `devtools` doesn't have an encrypted `GITHUB_PAT` env var on Travis yet.

In reality, I forget and have to make even more commits :(